src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -25,15 +25,22 @@ import (
 	"sync"
 	"time"
 
+	"github.com/hashicorp/go-multierror"
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
 	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/net"
 )
 
+// ErrNoAdminAPILeader happen when there's no leader for the Admin API
+var ErrNoAdminAPILeader = fmt.Errorf("no Admin API leader found")
```

Comment:
`= errors.New("no admin API leader found")`, since this is not formatting anything